cppcache/src/Cache.cpp

```diff
@@ -153,7 +153,12 @@ LogLevel Cache::getLogLevel() { return Log::logLevel(); }
 
 PdxInstanceFactory Cache::createPdxInstanceFactory(
     const std::string& className) const {
-  return m_cacheImpl->createPdxInstanceFactory(className);
+  return createPdxInstanceFactory(className, true);
```

Comment:
Why does this default to `true`, wouldn't it have been implicitly `false` perviously?

Reply:
By default a Java class is expected, that has always been the case. I get it can be a little bit confusing as flag in the serialized PdxType is inverted.